src/Params.ml

```diff
@@ -113,52 +113,112 @@ let get_record_type (c : Conf.t) =
       ; break_after= break space (-2)
       ; docked_after= fmt "}" }
 
-type record_expr =
+type elements_collection =
   { box: Fmt.t -> Fmt.t
-  ; break_after_with: Fmt.t
   ; sep_before: Fmt.t
   ; sep_after_non_final: Fmt.t
   ; sep_after_final: Fmt.t }
 
+type elements_collection_record_expr =
+  {common: elements_collection; break_after_with: Fmt.t}
```

Comment:
I'm not fan of the `common` field. It is defined again for record pat. What do you think of removing it from `elements_collection_record_expr` and returning `elements_collection * elements_collection_record_expr` from `get_record_expr` ?

Reply:
done